src/main/java/org/usfirst/frc4904/auton/JustMove.java

```diff
@@ -0,0 +1,11 @@
+package org.usfirst.frc4904.auton;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisMoveDistance;
+
+class JustMove extends AutonRoutine {
+
+    public JustMove() {
+        this.andThen(new ChassisMoveDistance(RobotMap.Component.chassis, 10, null)); // TODO: what motioncontroller?
```

Comment:
why not just use `SimpleSplines`?

Reply:
No need to if the robot is just moving forwards?